runtime/trappist/src/constants.rs

```diff
@@ -16,8 +16,8 @@
 // limitations under the License.
 
 pub mod currency {
-	use kusama_runtime_constants as constants;
 	use polkadot_core_primitives::Balance;
+	use rococo_runtime_constants as constants;
```

Comment:
Seemingly innocuous change ... but, if I'm not mistaken, it does change the scale of the currency units 😬️ (`CENTS`, `GRAND`, `MILLICENTS`). Please double-check.

Reply:
`rococo_runtime_constants::currency::UNITS` = 1_000_000_000_000

`rococo_runtime_constants::currency::UNITS` = 1_000_000_000_000

They are the same.

Reply:
Oh but you are totally right, the other constants are different!

Fixed in [88a0b6a](https://github.com/paritytech/trappist/pull/315/commits/88a0b6a5a8a246bf304a0f815d53f372c71b8323).

Very nice catch!

Reply:
🤓️😉️